model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -53,43 +56,67 @@ private fun createCopyrightFinding(statement: String, path: String, line: Int =
         )
     )
 
+private fun Collection<LicenseFindings>.getFindings(license: String) = single { it.license == license }
```

Comment:
Why not define this function inside of the `FindingsMatcherTest` class?

Reply:
Decision tree:
-class instance scope? -> no, no member vars needed
-companion object? -> bit boiler plate, discoverability not needed, it's better to not see it when reading the test class 
-top level: no drawbacks

What do you think?

Reply:
Makes sense.